src/internal/dockertestenv/util.go

```diff
@@ -94,7 +94,7 @@ func ensureContainer(ctx context.Context, dclient docker.APIClient, containerNam
 	hostConfig := &container.HostConfig{
 		PortBindings: portBindings,
 	}
-	resp, err := dclient.ContainerCreate(ctx, containerConfig, hostConfig, nil, containerName)
+	resp, err := dclient.ContainerCreate(ctx, containerConfig, hostConfig, nil, nil, containerName)
```

Comment:
I assume not setting the platform requests the host OS and Architecture.
https://pkg.go.dev/github.com/docker/docker/client#Client.ContainerCreate

If the INTERNAL tests are passing then it's probably fine.

Reply:
I wasn't really sure here, but yeah things seem to be okay? I can also dig in a bit more